dao/mongo.go

```diff
@@ -131,11 +127,39 @@ func (m *MongoService) CreatePractitionersResource(dao *models.PractitionerResou
 		"$set": insolvencyResource,
 	}
 
-	_, err = collection.UpdateOne(context.Background(), bson.M{"links.transaction": "/transactions/" + transactionID}, update)
+	_, err = collection.UpdateOne(context.Background(), filter, update)
 	if err != nil {
 		log.Error(err)
 		return fmt.Errorf("there was a problem handling your request for transaction %s", transactionID), http.StatusInternalServerError
 	}
 
 	return nil, http.StatusCreated
 }
+
+func (m *MongoService) GetPractitionerResources(transactionID string) ([]models.PractitionerResourceDao, error) {
```

Comment:
Need to add a comment for this method and the CreatePractitionersResource method similar to the comment on line 67